src/bitcoin-cli.cpp

```diff
@@ -1027,7 +1053,31 @@ static void ParseGetInfoResult(UniValue& result)
         }
         result_string += "\n";
     }
-
+    
+    result_string += strprintf("Feerate Distribution: \n");
```

Comment:
Maybe add color to standardize with the other headers?

Reply:
Changed 'Feerate Distribution' to 'Mempool' and added RED color